src/networkProviders/apiNetworkProvider.ts

```diff
@@ -149,6 +149,12 @@ export class ApiNetworkProvider implements INetworkProvider {
         return transaction;
     }
 
+    async getTransactions(address: Address): Promise<TransactionOnNetwork[]> {
```

Comment:
As opposed to the python implementation, here we don't support options? Shouldn't we?

Reply:
We can add it in the TODO list. If someone needs to do a custom request, they can use `doGetGeneric()`.